src/components/home/index.css

```diff
@@ -0,0 +1,22 @@
+/* cards style */
+.cardsContainer {
```

Comment:
i think you want a more specific classname here. what if your teammate also has something that has cards in it? then both of them are called `.cardsContainer`, no one knows which card it refers to.